### PR TITLE
#5 Added service for images extraction from PDF

### PR DESCRIPTION
**Pull request by ron4548**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,11 +3,13 @@ import {RouterModule, Routes} from '@angular/router';
 import {AppComponent} from './app.component';
 import {UserLoginComponent} from './users/user-login/user-login.component';
 import {UserProfileComponent} from './users/user-profile/user-profile.component';
+import {PdfjsTestComponent} from './pdfjs-test/pdfjs-test.component';
```
Comment (menhel) [inline src/app/app-routing.module.ts]: We shouldn't include test files and test code in the development branch. Please remove all test associated code and files.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -13,13 +13,16 @@ import { UserProfileComponent } from './users/user-profile/user-profile.componen
 import {AngularFirestoreModule} from '@angular/fire/firestore';
 import { LogoComponent } from './logo/logo.component';
 import {AngularFontAwesomeModule} from 'angular-font-awesome';
+import { PdfjsTestComponent } from './pdfjs-test/pdfjs-test.component';
```
Comment (menhel) [inline src/app/app.module.ts]: remove this

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -3,11 +3,13 @@ import {RouterModule, Routes} from '@angular/router';
...
 
 const routes: Routes = [
   { path: '', redirectTo: '/profile', pathMatch: 'full'},
   { path: 'login', component: UserLoginComponent},
-  { path: 'profile', component: UserProfileComponent}
+  { path: 'profile', component: UserProfileComponent},
+  { path: 'pdf', component: PdfjsTestComponent }
```
Comment (menhel) [inline src/app/app-routing.module.ts]: remove this

File: src/app/app.module.ts [hunk 4]
```diff
@@ -13,13 +13,16 @@ import { UserProfileComponent } from './users/user-profile/user-profile.componen
...
+import {PdfService} from './pdf.service';
 
 @NgModule({
   declarations: [
     AppComponent,
     UserLoginComponent,
     UserProfileComponent,
-    LogoComponent
+    LogoComponent,
+    PdfjsTestComponent
```
Comment (menhel) [inline src/app/app.module.ts]: remove this

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (ron4548) [reply src/app/app-routing.module.ts]: Fixed

**Pull request merged**